code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -283,7 +283,7 @@
 
 /obj/item/weapon/nullrod/whip
 	name = "holy whip"
-	desc = "What a terrible night to be on Space Station 13."
+	desc = "What a terrible night to be on the Cyberiad!"
```

Comment:
@uraniummeltdown trying to find a good way to make this work, but the issue is, it needs to be a constant.